README.md

```diff
@@ -80,6 +80,29 @@ Old: 1
 New: 4
 ```
 
+## Groups
+
+`locksmithd` coordinates the reboot lock in groups of machines. The default
+group is "", or the empty string. `locksmithd` will only coordinate the reboot
+lock with other machines in the same group.
+
+The purpose of groups is to allow faster updating of certain sets of machines
+while maintaining availability of certain services. For example, in a cluster
+of 5 CoreOS machines, if you have 2 load balancers and run `locksmithctl set-max 2`,
+then it is possible that both load balancers would be rebooted at the same time,
+interrupting the service they provide. However, if the load balancers are put
+into their own group named "lb", the other 3 machines will remain in the default
```

Comment:
Isn't clear: is set-max 2 per group or cluster-wide? If per group and you did set-max 2 in the default group would it then be possible for three hosts to reboot at the same time?